code/modules/surgery/organs/tongue.dm

```diff
@@ -397,9 +397,9 @@
 	desc = "A sophisticated ethereal organ, capable of synthesising speech via electrical discharge."
 	icon_state = "electrotongue"
 	say_mod = "crackles"
+	taste_sensitivity = 10 // ethereal tongues function (very loosely) like a gas spectrometer: vaporising a small amount of the food and allowing it to pass to the nose, resulting in more sensitive taste
```

Comment:
this is cringe. I dont' see why this weird explanation is needed?

Reply:
I just thought it was neat

Reply:
lore posters

Reply:
It's a comment, get over it

Reply:
its a comment with lore definition